docs/docs/welcome.mdx

```diff
@@ -0,0 +1,90 @@
+
+![Nebari overview sequence](/img/nebari_overview_sequence.png)
+
+### Components
+
+The technology stack is an integration of the following existing open source libraries:
+
+- [**Terraform**](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.
+- [**Kubernetes**](https://kubernetes.io/docs/home/): a cloud-agnostic orchestration system
+- [**Helm**](https://helm.sh/): a package manager for Kubernetes
+- [**JupyterHub**](https://jupyter.org/hub): a shareable compute platform for data science
+- [**JupyterLab**](https://jupyterlab.readthedocs.io/en/stable/): a web-based interactive development environment for Jupyter Notebooks
+- [**Dask**](https://docs.dask.org/en/latest/): a scalable and flexible library for parallel computing in Python
+  - [**Dask-Gateway**](https://gateway.dask.org/): a secure, multi-tenant server for managing Dask clusters
+- [**Keycloak**](https://www.keycloak.org/): Open Source Identity and Access Management
+- [**GitHub Actions**](https://docs.github.com/en/actions): a tool to automate, customize, and execute software development workflows in a GitHub repository.
+- [**traefik**](https://traefik.io/): for routing web/tcp traffic inside cluster
+  - [**traefik-forward-auth**](https://github.com/thomseddon/traefik-forward-auth): for single sign on and easy securing of web applications
+
+Amongst the newly created open source libraries on the tech stack are:
+
+- [**Jupyterhub SSH**](https://github.com/yuvipanda/jupyterhub-ssh): brings the SSH experience to a modern cluster manager.
+- [**Jupyter Videochat**](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.
+- [**Conda Store**](https://github.com/quansight/conda-store): serves identical conda environments and controls its life-cycle.
+- [**Conda Docker**](https://github.com/conda-incubator/conda-docker): an extension to the docker concept of having declarative environments that are associated with Docker images
+  allowing tricks and behaviour that otherwise would not be allowed.
+- [**VS Code codeserver**](https://github.com/cdr/code-server): built-in web editor tied to jupyterlab server
+
+### Integrations
+
+Nebari does all the heavy lifting of integrating with common services. 
+
+- [**Prefect**](https://www.prefect.io/) - workflow management
+- [**ClearML**](https://clear.ml/) - machine learning experiment tracking and analysis
+- [**Prometheus**](https://prometheus.io/) - cluster monitoring
+- [**Grafana**](https://grafana.com/) - cluster monitoring visualizations
+- [**CDS Dashboards**](https://cdsdashboards.readthedocs.io/) - shared dashboarding across users
+
+## Why use Nebari?
+
+Nebari provides enables teams to build their own scalable compute infrastructure with:
+
+- Easy installation and maintenance controlled by a single configuration file.
+- Autoscaling JupyterHub installation deployed on the Cloud provider of your choice.
```

Comment:
Maybe need to add a disclaimer or note of the cloud providers supported in case someone's choice is something like OVH